### DISCUSSION TITLE
stack

### DISCUSSION THREAD

**Discussion by samuel-404** [sanjay-kv/Open-source-Practice#1061] [Q&A] [answered]
what is the best stack for learning open source contribution

**Comment by irfanbroo** [comment, marked as answer]
Web devlopement: Frontend,Backend,Database
you can learn these three if you want to go for web developement
For Mobile App developement, you can go for Java,Kotlin and for ios Swift,Xcode
There are plenty of other stacks which you can explore

**Comment by Yaser-123** [comment]
A great stack for learning open-source contribution is JavaScript with Node.js, since it's widely used in open-source projects and has a large, active community.
